wlm/src/main/java/ai/djl/serving/wlm/ModelInfo.java

```diff
@@ -588,6 +609,143 @@ private String inferEngine() throws ModelNotFoundException {
         throw new ModelNotFoundException("Failed to detect engine of the model: " + modelDir);
     }
 
+    private boolean isPythonModel(String prefix) {
+        return Files.isDirectory(modelDir.resolve("MAR-INF"))
+                || Files.isRegularFile(modelDir.resolve("model.py"))
+                || Files.isRegularFile(modelDir.resolve(prefix + ".py"))
+                || Utils.getEnvOrSystemProperty("HF_MODEL_ID") != null
+                || Files.isRegularFile(modelDir.resolve("config.json"))
+                || prop.containsKey("option.s3url")
+                || prop.containsKey("option.model_id");
+    }
+
+    private String inferLMIEngine() throws ModelException, IOException {
+        // MMS/Torchserve
+        if (Files.isDirectory(modelDir.resolve("MAR-INF"))) {
+            return "Python";
+        }
+        JsonObject modelConfig = getHuggingFaceModelConfig();
+        if (modelConfig == null) {
+            return "Python";
```

Comment:
This would be the pure Python mode case where user is using DJLServing to run a generic python backend, which is valid. One scenario would be where user just provides a model.py file and maybe a .bin file for the checkpoint. 

In the `getHuggingFaceModelConfig` method, if we have a model_id (either s3url, option.model_id, or HF_MODEL_ID) but we can't load the config for that model_id we throw an exception. For that scenario I think throwing the exception makes sense since if we can't load the config, the python backend will also fail. But we could change that to generic python backend use case.